docs(linter): reformat doc comments

overlookmotel opened the pull request:
Really anal style nits for CLI options. Standardize formatting based on the following rules:

* Capitalized first letter.
* No full stops on end, unless comment contains multiple sentences (in which case, always full stop).
* Quote filenames and technical names in backticks (e.g. `.eslintignore`, `nursery`).
* Imperative style ("Disable" not "Disables").

These are the existing stylistic conventions, which comments mostly followed. This PR just corrects the few which didn't, to bring all the doc comments into line stylistically.

Main purpose is to clean up the docs on website: https://oxc.rs/docs/guide/usage/linter/cli.html

overlookmotel commented:
* **#15670** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15670?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15670?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.com/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=15670).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

camc314 commented:
### Merge activity

* **Nov 13, 1:49 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Nov 13, 1:49 PM UTC**: `camc314` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

overlookmotel committed:
docs(linter): reformat doc comments (#15670)

Really anal style nits for CLI options. Standardize formatting based on the following rules:

* Capitalized first letter.
* No full stops on end, unless comment contains multiple sentences (in which case, always full stop).
* Quote filenames and technical names in backticks (e.g. `.eslintignore`, `nursery`).
* Imperative style ("Disable" not "Disables").

These are the existing stylistic conventions, which comments mostly followed. This PR just corrects the few which didn't, to bring all the doc comments into line stylistically.

Main purpose is to clean up the docs on website: https://oxc.rs/docs/guide/usage/linter/cli.html
